content/doc/developer/forms/jelly-form-controls.adoc

```diff
@@ -6,10 +6,16 @@ references:
   title: ui-samples-plugin
 ---
 
-Most of the the jelly files in the Jenkins source have embedded documentation. See link:/doc/developer/development-environment/ide-configuration[IDE Configuration] for details on how to get this setup so you can click through to it in your IDE
+Most of the the jelly files in the Jenkins source have embedded documentation. See link:/doc/developer/development-environment/ide-configuration[IDE Configuration] for details on how to get this setup so you can click through to it in your IDE. 
+
+For example you can see the embedded documentation of link:https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/form.jelly[ the <form> tag ].
```

Comment:
Beware that floating branch links such as this can become invalid after refactorings.

Reply:
yes I don't know how to do it the best way, but I think the doc should link to jelly content as the documentation is live there.

Reply:
Changed